### ISSUE TITLE
Trend/SmartScalar visualization returns NaN when % growth over 1000

### ISSUE THREAD

**Issue by dacort** [metabase/metabase#11630]
**Describe the bug**
When calculating % growth in the Trend visualization, values over 1000% result in `NaN` being displayed.

**Logs**
No relevant console or server logs.

**To Reproduce**
1. Go to "Ask a question"
2. Select "Native query"
3. If necessary, select the "Sample Dataset"
4. Paste this query into the SQL editor:
    ```sql
    select `period`, sum(`count`)
    from (
    select timestamp '2019-10-01' as `period`, 1 as `count`
    union all
    select timestamp '2019-11-01' as `period`, 11 as `count`
    )
    group by `period`
    order by `period`
    ```
5. Run query
6. Click on visualization button, select "Trend"
7. Note that visualization shows "NaN"
8. Change the `11` above to something smaller (`8`) and re-run the query
9. Note that visualization now shows "700%"

**Expected behavior**
% growth over 1000 should show the correct value.

**Screenshots**
 Screenshot of error state:
![image](https://user-images.githubusercontent.com/1512/71746819-d5442480-2e22-11ea-8542-bd5c4b1841e2.png)

**Information about your Metabase Installation:**

Tested on Metabase v0.32.10, v.0.33.6, and v0.34.0

```json
{
  "browser-info": {
    "language": "en-US",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.88 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.3+7",
    "java.vendor": "AdoptOpenJDK",
    "java.vendor.url": "https://adoptopenjdk.net/",
    "java.version": "11.0.3",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.3+7",
    "os.name": "Linux",
    "os.version": "4.9.184-linuxkit",
    "user.language": "en",
    "user.timezone": "GMT"
  },
  "metabase-info": {
    "databases": [
      "h2"
    ],
    "hosting-env": "unknown",
    "application-database": "h2",
    "run-mode": "prod",
    "version": {
      "date": "2019-12-19",
      "tag": "v0.34.0",
      "branch": "release-0.34.x",
      "hash": "1335215"
    },
    "settings": {
      "report-timezone": null
    }
  }
}
```

**Severity**
Annoying in cases where % growth is high.

**Additional context**
n/a

**Comment by dacort** [comment]
Looks like this is likely due to calling [`Math.abs`](https://github.com/metabase/metabase/blob/21cbb87a8604c4db462e002736b7508d1a9b200e/frontend/src/metabase/visualizations/visualizations/SmartScalar.jsx#L120) on the result of `formatNumber`, which is a string.

```javascript
Math.abs("123")
> 123
Math.abs("-123")
> 123
Math.abs("1,234")
> NaN
```

**Comment by sbelak** [comment]
Yes, I've double-checked and the result that comes from the server is correct.

**Comment by paulrosenzweig** [comment]
Whoops I missed this issue! It was fixed in #11649
